ios/FlankerNativeComponents/GameManager.swift

```diff
@@ -137,14 +137,14 @@ class GameManager {
 
   func checkedAnswer(button: SelectedButton) {
     guard !hasRespondedInCurrentTrial else { return }
+    guard let startTrialTimestamp = startTrialTimestamp else { return }
```

Comment:
With this change I'm encountering a `index out of range` error in `checkAnswer` at line 157
`let correctChoice = gameParameters.trials[countTest].correctChoice`

![CleanShot 2024-11-22 at 09 15 56](https://github.com/user-attachments/assets/3d3193a1-2c1b-4bf1-b64f-8057d1a90220)

Reply:
Could you test again ? I built several times and I didnt get this error but even so I added a line to prevent this error. 
let me know if it is working.

Reply:
I was unable to reproduce, but I see the fix you added and makes sense for the issue I encountered